src/util-magic.c

```diff
@@ -120,11 +120,13 @@ char *MagicThreadLookup(magic_t *ctx, const uint8_t *buf, uint32_t buflen)
 
 #ifdef UNITTESTS
 
+#if 0
```

Comment:
Why this change ?

I see `MICROSOFT_OFFICE_DOC` getting used

Reply:
I see a few references but they're in functions that are within `#if 0` blocks.